index.html

```diff
@@ -0,0 +1,127 @@
+                    Lorem ipsum, dolor sit amet consectetur adipisicing elit. Dicta dolor sequi optio necessitatibus quis praesentium, inventore adipisci distinctio obcaecati voluptatum provident sapiente illum earum ullam deserunt deleniti consequuntur reprehenderit cupiditate. Lorem ipsum dolor sit amet, consectetur adipisicing elit. In quo modi accusamus similique vitae sed animi. Hic cumque officia alias cupiditate eos aspernatur suscipit! Necessitatibus error suscipit vel hic repudiandae! Lorem ipsum dolor, sit amet consectetur adipisicing elit. Debitis illo amet eum facere eius, porro sed assumenda quisquam sint reiciendis quae quidem minus. Id temporibus officia unde quaerat praesentium quam. Lorem, ipsum dolor sit amet consectetur adipisicing elit. Pariatur iste architecto impedit, eligendi nisi eveniet, expedita corporis voluptatum nesciunt nulla non reprehenderit ab maxime exercitationem officiis aperiam, quidem porro recusandae?</p>
+            </div>
+          </div>   
+    </section>
+    <br>
+    <br>
+    <br>
+    <br>
+    <section id="Premiun"><center><h1>PREMIUN CONTENT</h1></center>
+
+        <div class="row">
+            <div class="column">
+                <center><h2>Disney+</h2>
+                <img id="section-img" src ="https://img.icons8.com/nolan/2x/disney-plus.png"></center>
+            </div>
+            <div class="column">
+                <center><h2>HBO</h2>
+                <img id="section-img" src ="https://cdn.iconscout.com/icon/free/png-128/hbo-2-555595.png"></center>
+            </div>
+            <div class="column">
+                <center><h2>AMAZON PRIME</h2>
+                <img id="section-img" src ="https://th.bing.com/th/id/R.ecca3f0cb3bc20302abe55f83290a929?rik=ZQFFjPrUrrXQZA&riu=http%3a%2f%2fwww.newdesignfile.com%2fpostpic%2f2014%2f10%2famazon-com_68234.png&ehk=aT76FZZIx9UoLeAgmtM%2fuxv%2bIkTol%2frhP3Jw70i3X4k%3d&risl=&pid=ImgRaw&r=0"></center>
+            </div>
+          </div>
+    
+
+    
+
+    
+    </section>
+    <br>
+    <bR>
+    <br>
+    <br>
+    <br>
+    <section id="Support"><center><h1>TECHNICAL SUPPORT</h1></center>
+
+    <center><iframe id="video" width="560" height="315" src="https://www.youtube.com/embed/p52PY_cwIsA" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+      </iframe>
+      <form id="form" action=" https://www.freecodecamp.com/email-submit)">  
+        <P>Send us your email for more info....</P>
+        <input type="email" id="email" name="email" placeholder="Write Your Email">
+        <input id="submit" type="submit" value="More info"> </center></form>
+    </section>
+
+        
+    
+    
+</body>
+<script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+<script  src="js/index.js"></script>
+</html>
```

Comment:
As I mentioned before, please make sure you use justify-content instead of using a center tag and not leave too much spaces between code.

Reply:
As I mentioned before, please make sure you use justify-content instead of using a center tag, and be careful with blank lines and indentation for writing more readable and clean code.